app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -553,6 +562,32 @@ def alert_valid_record?(alert)
         end
       end
     end
+    if ["mw_heap_used", "mw_non_heap_used"].include?(@edit.fetch_path(:new, :expression, :eval_method))
+      value_greater_than = @edit.fetch_path(:new, :expression, :options, :value_mw_greater_than)
+      non = @edit.fetch_path(:new, :expression, :eval_method) == "mw_non_heap_used" ? "Non" : ""
+      an_integer = "an integer"
+      between = "between 0 and 100"
+      template_error = "%s %s Heap Max (%%) must be %s"
+      unless value_greater_than && is_integer?(value_greater_than)
+        add_flash(_(template_error % [">", non, an_integer]), :error)
+      end
+      unless value_greater_than.to_i.between?(0, 100)
+        add_flash(_(template_error % [">", non, between]), :error)
+      end
+      value_less_than = @edit.fetch_path(:new, :expression, :options, :value_mw_less_than)
+      unless value_less_than && is_integer?(value_less_than)
+        add_flash(_(template_error % ["<", non, an_integer]), :error)
+      end
+      unless value_less_than.to_i.between?(0, 100)
```

Comment:
@abonas I definitely understand what you're saying with the error message looking like it was messed up by the system because of the random (%) in the middle of it, but I think changing that to the word percent would also change the label names on the form.
![image](https://cloud.githubusercontent.com/assets/3739227/16380620/256cf7b6-3c46-11e6-97cc-9c20a0143ab0.png)

I think having (%) next to the label forms makes it clearer to the user at a glance that we're asking for a % number in the field.  Would you agree?